integ-tests/src/main/java/com/google/pojo/UserTableRow.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.pojo;
+
+public class UserTableRow {
+
+    public UserName userName = new UserName();
```

Comment:
Please use google's Java formatting. You can use `google-java-format` for this.

The following allows you to just run `git fix` to fix all files impacted by the current commit:

```
git config --global alias.fix \
  '!git show --pretty="" --name-only HEAD | while read -r f || [[ -n "${f}" ]]; do google-java-format -i "${f}"; done'
```

Reply:
Ok, I've done this. however the formatter didn't change exactly this place but some others. Is that ok or you want me to delete the line 19 and other like it?

Reply:
The git command only fixes HEAD. So, it might not have worked on all files. As you can see in the other files, we use two space indents.

My point was not to delete line 19, but the indenting. So, maybe run the google formatter on all files you added.

Reply:
Will do.